stdlib/format.mli

```diff
@@ -518,6 +515,7 @@ val safe_set_geometry : max_indent:int -> margin:int -> unit
    Outside of this domain, [pp_set_geometry] raises an invalid argument
    exception whereas [pp_safe_set_geometry] does nothing.
```

Comment:
I think this comment isn't accurate with the explicit upper bound anymore. As shown in #11517, `pp_set_geometry` (not safe) doesn't raise when the values are too large.

Reply:
You are right. Thank you! I updated the docs to be more explicit about when the exception is raised. 

What do you think?